packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -89,10 +89,12 @@
         justify-content: space-between;
         margin: 2.4em 0;
         padding: 1.6rem;
+        width: 97.5%;
```

Comment:
Why not calculate with offset instead? Percentage values are relative to container size, it's better to not use fixed percentage values.

```Example
        width: calc(100% - {offset_here_rem_or_px});
```